### PR TITLE
Eda 886

### PR DESCRIPTION
**Pull request by sarias-eb**
As we talked earlier, I add the changes from BankAccount

### CODE REVIEW

File: supplier_app/migrations/0021_auto_20200504_1752.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from django.db import migrations, models
+import django.db.models.deletion
+import simple_history.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('supplier_app', '0020_auto_20200430_1226'),
```
Comment (axelfernandez-eb) [inline supplier_app/migrations/0021_auto_20200504_1752.py]: This make conflict!

File: supplier_app/migrations/0021_auto_20200505_2031.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('supplier_app', '0020_auto_20200430_1226'),
```
Comment (axelfernandez-eb) [inline supplier_app/migrations/0021_auto_20200505_2031.py:12]: This too!

File: supplier_app/migrations/0021_auto_20200504_1752.py (hunk 1 above)
Comment (sarias-eb) [reply supplier_app/migrations/0021_auto_20200504_1752.py]: Thanks :)

File: supplier_app/migrations/0021_auto_20200505_2031.py (hunk 2 above)
Comment (sarias-eb) [reply supplier_app/migrations/0021_auto_20200505_2031.py:12]: Another thanks 👍

**Pull request merged**